hg-fast-export.py

```diff
@@ -28,6 +28,9 @@
 # write some progress message every this many file contents written
 cfg_export_boundary=1000
 
+subrepo_cache = {}
```

Comment:
Most of the code-base does not (except for where I have slipped up and let patches through) use spaces around `=` and '=='.

Reply:
Fixed!